src/rust/.sqlx/query-5134297b8b3f2ebe0add9948c4588a67170ef60b22d1cb8e608b71b9fe883ac1.json

```diff
@@ -0,0 +1,16 @@
+{
+  "db_name": "PostgreSQL",
+  "query": "\n            -- Insert into the users table\n            INSERT INTO aggregator.competition_leaderboard_users (\"user\", volume, integrators_used, n_trades, competition_id)\n\n            -- Every address with default values from new fill events\n            SELECT DISTINCT \"user\", 0, '{}'::text[], 0, $1 FROM aggregator.current_fills($1,$2,$3)\n            -- That doesn't already exist\n            WHERE NOT EXISTS (\n                SELECT *\n                FROM aggregator.competition_leaderboard_users\n                WHERE competition_id = $1\n                AND current_fills.\"user\" = aggregator.competition_leaderboard_users.\"user\"\n            )\n            UNION\n            -- Every address with default values from new place events\n            SELECT DISTINCT \"user\", 0, '{}'::text[], 0, $1 FROM aggregator.current_places($1,$2,$3)\n            -- That doesn't already exist\n            WHERE NOT EXISTS (\n                SELECT *\n                FROM aggregator.competition_leaderboard_users\n                WHERE competition_id = $1\n                AND current_places.\"user\" = aggregator.competition_leaderboard_users.\"user\"\n            )\n        ",
```

Comment:
How about taking out this change?

This PR doesn't affect SQLx

Reply:
done